heareval/predictions/task_predictions.py

```diff
@@ -865,6 +874,18 @@ def _combine_target_events(split_names: List[str]):
         validation_target_events: Dict = _combine_target_events(data_splits["valid"])
         test_target_events: Dict = _combine_target_events(data_splits["test"])
 
+        # The postprocessing search space for getting the
+        # best task specific postprocessing, can be task
+        # specific, present in the task metadata in
+        # evaluation_params.postprocessing_grid. If not, the default
+        # EVENT_POSTPROCESSING_GRID will be used.
+        if "event_postprocessing_grid" in metadata["evaluation_params"]:
+            postprocessing_grid = metadata["evaluation_params"][
+                "event_postprocessing_grid"
+            ]
+        else:
+            postprocessing_grid = EVENT_POSTPROCESSING_GRID
```

Comment:
EVENT_POSTPROCESSING_GRID can we just remove this and have a specific dcase grid and no default? If this is tricky let's just make an issue for it.

Reply:
#332